### PR TITLE
nvuApp: Logout button is set up

### PR DESCRIPTION
**Pull request by qnislavnachev**
on user and admin home page is added a logout button that kills
session and cookie

### CODE REVIEW

**Comment by qnislavnachev** [comment]
logout button is added to home page of user and admin

File: src/main/resources/http/servlet/adminHome.html [hunk 1]
```diff
@@ -6,7 +6,14 @@
     <link rel="stylesheet" type="text/css" href="/assets/home.css">
 </head>
 <body>
-<h1>Добре дошли.</h1>
+<div>
+    <h1>Добре дошли.</h1>
+    <div style="text-align: right">
+        <form action="/logout">
```
Comment (mgenov) [inline src/main/resources/http/servlet/adminHome.html]: You don't need a form. Simple link could do the job.

File: src/test/java/com/clouway/nvuapp/core/LogoutHandlerTest.java [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+import persistent.dao.DataStore;
+
+import javax.servlet.http.Cookie;
+import java.util.Optional;
+
+import static core.CustomMatchers.isDead;
+import static core.CustomMatchers.isNotPresent;
+import static org.junit.Assert.assertThat;
+
+public class LogoutHandlerTest {
```
Comment (mgenov) [inline src/test/java/com/clouway/nvuapp/core/LogoutHandlerTest.java]: logoutWhenSessionIsNotAvailable case is not handled.

**Comment by qnislavnachev** [comment]
added test case and used a link instead a form in html files

**Pull request merged**